src/Validation/Validation.php

```diff
@@ -1431,6 +1431,17 @@ public static function isArray($value)
     }
 
     /**
+     * Check that the input value is a 6 digits hex color.
```

Comment:
The context here was validating html input type=color and I could not find a quick answer if that supports 3 digits or only 6 digits.

As a general purpose rule it should probably support both and have a flag to toggle: `'digits' => ['both', '3', '6']` and then be set to 6 (or both) for input type color.